configuration/cdap-site.xml

```diff
@@ -1047,6 +1082,139 @@
     </value-attributes>
   </property>
 
+  <!-- Messaging System Configuration -->
+
+  <property>
+    <name>messaging.container.instances</name>
+    <value>1</value>
+    <description>
+      Number of instances for the messaging service
+    </description>
+    <final>true</final>
+  </property>
+
+  <property>
+    <name>messaging.container.memory.mb</name>
+    <value>${master.service.memory.mb}</value>
+    <description>
+      Memory in megabytes for each messaging service instance
+    </description>
```

Comment:
I added the elements from log.saver.container.memory.mb but because we added a type of int it doesn't retain the default value of ${master.service.num.cores}